Firebase/ios/Bonfire/Mobile/Login/LoginView.swift

```diff
@@ -21,17 +58,36 @@ final class LoginView: UIView {
     }
 
     func setupViews() {
-        backgroundColor = .whiteColor()
+        backgroundColor = BonfireColors.orange
+
+        disclaimerLabel.text = "This application is for demo purposes only. Signing in via your Google account will give us access to your name and profile image only, which will be publicly available to all other users of the application. The messages are not encrypted and any Novoda employee can potentially see them. The database, including all channels and messages can be erased at anytime."
+        disclaimerLabel.textAlignment = .Center
+        disclaimerLabel.numberOfLines = 0
+        disclaimerLabel.font = UIFont.systemFontOfSize(12)
+        disclaimerLabel.textColor = .whiteColor()
+
         googleButton.style = .Wide
+
+        bubblesBackground.updateWithView(logoView)
     }
 
     func setupLayout() {
+        addSubview(bubblesBackground)
+        addSubview(disclaimerLabel)
         addSubview(googleButton)
 
-        googleButton.pinToSuperviewTop(withConstant: 20)
+        bubblesBackground.pinToSuperviewTop()
+        bubblesBackground.pinToSuperviewLeading()
+        bubblesBackground.pinToSuperviewTrailing()
+        bubblesBackground.pinToSuperviewBottom()
+
+        disclaimerLabel.pinToSuperviewLeading(withConstant: 25)
```

Comment:
If they were more consistent, maybe. Not sure enough are reused. But if we were using standard/default margins in our designs, I'd agree